docs/generated/packages/jest/executors/jest.json

```diff
@@ -70,7 +70,8 @@
       },
       "ci": {
         "description": "Whether to run Jest in continuous integration (CI) mode. This option is on by default in most popular CI environments. It will prevent snapshots from being written unless explicitly requested. (https://jestjs.io/docs/cli#--ci)",
-        "type": "boolean"
+        "type": "boolean",
+        "x-priority": "important"
```

Comment:
unless there's some specific reason here, I think `ci` shouldn't be `important` either. it is set by default and kind of a generic option.